integrations/SnapPixel/util.js

```diff
@@ -42,21 +47,39 @@ const getCommonEventPayload = (message) => {
   if (payload.success !== 0 && payload.success !== 1) {
     payload.success = null;
   }
+  if (enableDeduplication) {
+    payload.client_deduplication_id = get(
+      message,
+      `${deduplicationKey || "messageId"}`
```

Comment:
why we are looking for `deduplicationKey` key name (from webapp) at the root level? Does it fall in our spec?

Reply:
`deduplicationKey` will be containing values like `properties.dedupKey` or `context.traits.dedupKey`

Reply:
If we go with this structure then we should update all other destinations too which have this feature in order to align with the design so that the user does not face any confusion while using this field if someone has set up multiple destinations at their end.

Reply:
We will .. one step at a time man

Reply:
Pointed out in order to keep note during updation